dev/Repeater/ItemsRepeater.idl

```diff
@@ -334,12 +334,14 @@ unsealed runtimeclass ElementAnimator
     protected void OnBoundsChangeAnimationCompleted(Windows.UI.Xaml.UIElement element);
 }
 
-[WUXC_VERSION_PREVIEW]
+[WUXC_VERSION_MUXONLY]
 [webhosthidden]
```

Comment:
Let's hold off on doing this until we've got API review signoff on the name

Reply:
Yes. That is the plan. I also want to add a couple of tests to cover the new property exposed.